src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 29px 14px 14px;
+
+  --def-font-size: 11px;
```

Comment:
You could just move 'font-size: 11px' to body and it would be applied everywhere as default font size and not repeat yourself three times